src/elements/content-preview/ContentPreview.js

```diff
@@ -1113,11 +1123,18 @@ class ContentPreview extends React.PureComponent<Props, State> {
      */
     onVersionChange = (version?: BoxItemVersion, additionalVersionInfo: AdditionalVersionInfo = {}): void => {
         const { onVersionChange }: Props = this.props;
+        const { location } = additionalVersionInfo;
+
         this.updateVersionToCurrent = additionalVersionInfo.updateVersionToCurrent;
 
         onVersionChange(version, additionalVersionInfo);
+
         this.setState({
             selectedVersion: version,
+            startAt: {
+                unit: location.type === 'page' ? 'pages' : undefined,
+                value: location.value,
```

Comment:
Does the Preview SDK handle these properties being undefined if the `startAt` object is defined? Maybe we should make the whole `startAt` object undefined or null?

Reply:
It looks like it does, but I think you're right, it's safer to make the whole object undefined